### ISSUE TITLE
Can i put my own images inside this Guidebook? (not from MC resourses)

### ISSUE THREAD

**Issue by F-L-O-K-I** [gigaherz/Guidebook#117]
If i can - where should i store it? 
Mb its real with some other mods?

**Comment by gigaherz** [comment]
Yes, all images you have in resources in the config folder can be used.
That is, you can put a PNG image in your config folder like this:
```
config/
    books/
        resources/
            textures/
                image1.png
        book1.xml
```
and then you can reference it as `"gbook:image1"`

**Comment by gigaherz** [comment]
Oh forgot to say, if you are making a resource pack for the book you can just have the images in the resource pack, and then you can use whatever namespace the resource pack is in, "gbook" or otherwise.

**Comment by F-L-O-K-I** [comment]
What am i doing wrong? Q_Q
(i'm trying to make resourcepack and add my own cover for book (48x48))

- I copyed "demobook" in same folder. 
- Renamed it "bestiary".
- add "gbook:books/bestiary.xml" ] in book.json
- Changed mcmeta to 6 (i'm on 1.16.5).
//from this moment i can see new book with the same name in the game:
![image](https://github.com/gigaherz/Guidebook/assets/139173103/73741691-0e43-4f9d-adc2-6e52f895ecf9)
__________________________________________
then i find this issue https://github.com/gigaherz/Guidebook/issues/40 and tryed to make new cover:

- Creating new folder "textures" in <assets/gbook>
- adding my own picture 48x48pix in this folder
- changing in bestiary.xml [<book title="Bestiary" cover="gbook:textures/bestiary" fontSize="0.6">]
-got this...
![image](https://github.com/gigaherz/Guidebook/assets/139173103/578ca1b5-89c5-4035-986d-a6751ceca833)

It is trouble with picture or i made smthing wrong?

**Comment by F-L-O-K-I** [comment]
Btw if i will change default.png in mod folders, on my own - all will be fine:
![image](https://github.com/gigaherz/Guidebook/assets/139173103/4cd78987-5ec1-4694-8dd5-335368dfeb9c)

**Comment by F-L-O-K-I** [comment]
I fix it! (not for resourcepack, but it works xD)
So, if u want add any custom picture in this mod (1.16.5) use this inissue guide:

This works ony if u creating new books in gbook folder. If you created other folder for own book - use it. (c)Cpt Logic

**I want add covers** 
- add it in `Guidebook(your version).jar\assets\gbook\textures\covers` folder in mod repository
- add link on your custom picture in .xml book with: `cover="gbook:covers/ _custom picture name without .png_`"
**I want add my own pictures in books** 
- create new folder in `gbook\textures\`. For example i call it images, so now i have `Guidebook(your version).jar\assets\gbook\textures\images`
- put your best .png in this folder, so u can link them from .xml with: `<image src="gbook:images/ _your picture name without .png_" tw="16" th="16"/>`
- add your picture size in last 2 parameters (tw - width, th - height)

Ghz, sry cuz bothering you with stuped questions, but this awesome mod really need detailed wiki or better videoGuide!
I believe, one day, someone will create it!)

**Comment by gigaherz** [comment]
Uh you are not supposed to edit the .jar file! That will not work if you are making a modpack to share with others via curseforge or such!
I haven't had time to look into your issue yet, but it should work from the config folder, or using the same paths in a resource pack.
